### DISCUSSION TITLE
pycryptodomex

### DISCUSSION THREAD

**Discussion by agieocean** [pyodide/pyodide#4523] [Q&A]
I see there's a built in package for pycryptodome but the wheel I'm importing requires pycryptodomex. I've tried building the wheel for it but that just generates a wheel for pycryptodome. How can I work around this?

**Comment by hoodmane** [comment]
It looks like @Starz0r added pycryptodomex in #2966 but it never worked and was removed in #3261.

**Comment by ryanking13** [comment]
Maybe you can start by downloading the pycryptodomex sdist (.tar.gz) from pypi, and build from there?
